lib/Expr/Expr.cpp

```diff
@@ -705,7 +708,6 @@ ref<Expr> NotExpr::create(const ref<Expr> &e) {
   return NotExpr::alloc(e);
 }
 
```

Comment:
Remove extra line

Reply:
Which extra line? 
I can't see anything in the format of a green line with a '+' in the beginning (The way GitHub shows new lines added).